source/server/setup/setup-service-configurations.js

```diff
@@ -0,0 +1,23 @@
+let env = Space.getenv.multi({
+  loginStyle: ['SPACE_ACCOUNTS_SERVICE_CONFIGURATION_LOGIN_STYLE', 'popup', 'string'],
+  googleClientId: ['SPACE_ACCOUNTS_SERVICE_CONFIGURATION_GOOGLE_CLIENT_ID', '', 'string'],
+  googleSecret: ['SPACE_ACCOUNTS_SERVICE_CONFIGURATION_GOOGLE_SECRET', '', 'string']
+});
+
+let setupGoogleServiceConfiguration = function() {
+
+  if (env.googleClientId !== '' && env.googleSecret !== '') {
```

Comment:
What's the impact of changing an existing `ServiceConfiguration.configurations` by providing new creds in the ENVs?

I was also going to suggest we move this config into the accounts module, or at least wrap it up as a Space.Object then link it as a Singleton. That way the module's lifecycle can be used rather than `Meteor.startup`, and the application's config used to enable external services during init